app/src/main/java/ch/sheremet/dasfoo/rover/android/client/property/PropertyReader.java

```diff
@@ -0,0 +1,37 @@
+package ch.sheremet.dasfoo.rover.android.client.property;
+
+import android.content.Context;
+import android.content.res.AssetManager;
+import android.util.Log;
+
+import java.io.InputStream;
+import java.util.Properties;
+
+/**
+ * Created by Katarina Sheremet on 6/10/16 2:48 PM.
+ */
+
+/** This class is created for temporary purpose. It will be deleted
```

Comment:
Please add a `TODO` for removing this class, makes it easier to build a list of things to take care of.